dgraph/graphql/src/api.js

```diff
@@ -1,29 +1,16 @@
 import { ApolloServer } from 'apollo-server-fastify';
 import fastify from 'fastify';
 import { typeDefs } from './schema';
-import { dgraph } from './data';
-
-const resolvers = {
-  Query: {
-    searchByName: async (_source, _args, { dataSources }) => {
-      const dgraphQuery = `{
-        query(func: regexp(description, /^${_args.text}/i)) @filter(has(code) AND eq(type, "unit_of_use")) {
-          code
-          description
-        }
-      }`;
-      const resp = await dataSources.dgraphDataSource.postQuery(dgraphQuery);
-      return resp.data.query;
-    },
-  },
-};
+import { DgraphDataSource, RxNavDataSource } from './data';
+import { resolvers } from './resolvers'
 
 const server = new ApolloServer({
   typeDefs,
   resolvers,
   dataSources: () => {
     return {
-      dgraphDataSource: new dgraph(),
+      dgraphDataSource: new DgraphDataSource(),
```

Comment:
Can probably just call this `dgraph`, `rxNav`?

Reply:
Hmm good point, I think we can call the variables here just dgraph and rxNav for brevity.
Updated with https://github.com/openmsupply/unified-codes/pull/72/commits/525700ac54637cac82059a19fd5760a92b41e00e